### PR TITLE
{geo}[foss/2022a] rasterstats v0.19.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/rasterstats/rasterstats-0.19.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+use_pip = True
+
+exts_list = [
+    ('affine', '2.3.1', {
+        'checksums': ['d676de66157ad6af99ffd94e0f54e89dfc35b0fb7252ead2ed0ad2dca431bdd0'],
+    }),
+    ('click', '8.1.3', {
+        'checksums': ['7682dc8afb30297001674575ea00d1814d808d6a36af415a82bd481d37ba7b8e'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/r/rasterstats/rasterstats-0.19.0-foss-2022a.eb:27]:
```suggestion
```

This is in [Python/3.10.4](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb#L339)

File: easybuild/easyconfigs/r/rasterstats/rasterstats-0.19.0-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ('cligj', '0.7.2', {
+        'checksums': ['a4bc13d623356b373c2c27c53dbd9c68cae5d526270bfa71f6c6fa69669c6b27'],
+    }),
+    ('simplejson', '3.17.6', {
+        'checksums': ['cf98038d2abf63a1ada5730e91e84c642ba6c225b0198c3684151b1f80c5f8a6'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/r/rasterstats/rasterstats-0.19.0-foss-2022a.eb:33]:
```suggestion
```

This is in [Python/3.10.4](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb#L428)

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/24464a73f0efa0a6aff46bb670b6623d for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/1d07e0468c9746c024a30298a56ebad7 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @WilleBell!

**Pull request merged**